packaging/nnstreamer.spec

```diff
@@ -926,6 +926,7 @@ cp -r result %{buildroot}%{_datadir}/nnstreamer/unittest/
 %{_includedir}/nnstreamer/nnstreamer_internal.h
 %{_includedir}/nnstreamer/nnstreamer_log.h
 %{_includedir}/nnstreamer/tensor_filter_single.h
+%{_libdir}/pkgconfig/nnstreamer-internal.pc
```

Comment:
plz update nnstreamer-dev-internal (https://github.com/nnstreamer/nnstreamer/blob/main/debian/nnstreamer-dev-internal.install)

Reply:
oh.. yes, we definitely need to update it.